homeworks/david.tereshchenko_DavidTereshchenko/5-logic/homework.js

```diff
@@ -0,0 +1,55 @@
+const operator = document.querySelector('.operator');
+const btn = document.querySelector('.button');
+
+btn.onclick = function () {
+    const one = Number(document.getElementById('firstNumber').value);
+    const two = Number(document.getElementById('secondNumber').value);
+    const result = document.querySelector('.result');
+
+    let calc;
+
+    if (Number.isNaN(one) || Number.isNaN(two)) {
+        result.innerHTML = 'Enter a number';
+        return result;
+    }
+
+    if (one === 0 || two === 0) {
+        result.innerHTML = 'Enter a number';
+        return result;
```

Comment:
Your calculator doesn't calculate 0 at all. But it should. Usually, all calculations, except for division by 0, are performed on a calculator

Reply:
The result 'Enter a number' might be if a user typed a symbol or input field is empty